27-Toast Notifications/script.js

```diff
@@ -0,0 +1,36 @@
+const button = document.getElementById('button')
+const toasts = document.getElementById('toasts')
+
+const messages = [
+  'Message One',
+  'Message Two',
+  'Message Three',
+  'Message Four'
+]
+
+const types = ['info', 'succes', 'error']
+
+button.addEventListener('click', () => createNotification())
```

Comment:
hmmm, interesting, why not 
`button.addEventListener('click', createNotification)`?
what is the need for the wrapper arrow function?

Reply:
it learned after i put it ,  it  could of done the same way  you said ,
it's just when i learned that both way could work i just left it there  
i'll fix it .

Reply:
correct both ways work, it just that when you create an unnecessary function, that could be a room for inefficiency.